master/internal/usergroup/api_groups.go

```diff
@@ -61,8 +74,20 @@ func (a *UserGroupAPIServer) GetGroups(ctx context.Context, req *apiv1.GetGroups
 		return nil, apiutils.ErrInvalidLimit
 	}
 
-	groups, memberCounts, tableCount, err := SearchGroupsWithoutPersonalGroups(ctx,
-		req.Name, model.UserID(req.UserId), int(req.Offset), int(req.Limit))
+	query := SearchGroupsQuery(req.Name, model.UserID(req.UserId), true)
```

Comment:
I think we want ```includePersonalGroups``` to false since personal groups are an internal detail users shouldn't care about